clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/description/servergroup/DeployDcosServerGroupDescription.groovy

```diff
@@ -47,6 +52,7 @@ class DeployDcosServerGroupDescription extends AbstractDcosCredentialsDescriptio
     String type
   }
 
+  // TODO don't think we need a lot of this stuff (fromContext, fromTrigger, cluster, account)
```

Comment:
Yeah, I think orca uses them in the input map that comes in from deck, but when it gets to clouddriver I think we can take them out of this model.